master/buildbot/status/web/builder.py

```diff
@@ -191,29 +191,32 @@ def performAction(self, req):
         # send the user back to the builder page
         defer.returnValue(path_to_return)
 
+def buildForcePropertyName(scheduler, field):
+    return "%s.%s"%(scheduler.name, field.fullName)
+
 def buildForceContextForField(req, default_props, sch, field, master, buildername):
```

Comment:
while you are changing entire body here could you change `sch` to `scheduler`? It will be much easier to read in future.